book/website/project-design/data-governance/data-gov-ml.md

```diff
@@ -0,0 +1,74 @@
+The deep learning (DL) family of models, in particular, rely on massive corpuses of data such as text, code, images, sound, and other media.
+The process of data collection depends on the type and volume of data required and sources for acquisition.
+Many DL models rely on Internet data due to the sheer volume of digital content that is available on the web. 
+For example, [ImageNet datasets](https://www.image-net.org/about.php) are sourced from web images from image hosting websites like Flickr, and [LAION datasets](https://laion.ai/faq/) come from web crawling sources like Common Crawl.
+These methods of data collection through web scraping have raised issues regarding data quality and bias due to the nature of using uncurated Internet data, as well as issues of data rights as web scraped data sets often include copyrighted data or data obtained without consent that may be in violation of the data license and terms of use.
+To address issues related to respecting data licenses, some ML teams are choosing to only use open source or permissively licensed data.
+This can be challenging to accomplish with data that does not have explicit licenses, or for data scraped for the web which may not have the license included in the metadata.
+
+### Governance Tool: [Datasheets for Datasets](https://www.microsoft.com/en-us/research/uploads/prod/2019/01/1803.09010.pdf)
+
+Datasheets are resources accompanying datasets that can provide key information about the motivation to create the dataset, 
+the data collection process, the dataset composition and pre-processing, and legal and ethical considerations when using the data.
+Datasheets help contextualise the resource and are an opportunity for dataset creators to share any concerns, suggestions,
+and other advice regarding responsible use of the dataset.
+The [Datasheets for Datasets paper](https://www.microsoft.com/en-us/research/uploads/prod/2019/01/1803.09010.pdf) contains a list of example questions that data creators should ask themselves and document in their datasheet.
+
+(pd-dg-ml-dm)=
+## Data Management
+
+Once data collection is complete, challenges emerge around data storage, access, and making changes to the dataset.
+Because state-of-the-art ML datasets are very large, they tend to be stored in a central location, often a cloud provider database, and made available for download under certain circumstances.
+In addition to creating a governance structure for the opening or gating of access of the dataset to certain users, there is also the challenge of creating a governance structure to allow data creators to request to opt-out their data from the dataset, if the team is interested in providing that capability.
+Because opting out occurs after a dataset has already been created, it will not remove the data from original versions of the dataset, but can facilitate removal from future versions.
```

Comment:
Do we mean dataset or model here?

Reply:
Dataset! The thinking is that once a dataset exists you can't effectively "remove" data from it as it probably already exists/has been downloaded in multiple places

Reply:
Is a management service like [DagsHub](https://dagshub.com/) germane to this discussion?